_posts/en/newsletters/2021-01-27-newsletter.md

```diff
@@ -0,0 +1,116 @@
+mechanisms and includes our regular sections with top questions and
+answers from the Bitcoin StackExchange, a list of releases and release
+candidates, and descriptions of notable changes to popular Bitcoin
+infrastructure software.
+
+## News
+
+- **Scheduled meeting to discuss taproot activation:** Michael Folkson
+  [announced][folkson announce] that a meeting will be held at <time
+  datetime="2021-02-02 19:00-0000">19:00 UTC on February 2nd</time> in
+  the [##taproot-activation][] IRC channel on irc.freenode.net to
+  discuss some desired revisions to [BIP8][].  It has not yet been
+  decided that BIP8 will actually be used for the activation, so
+  alternative proposals may also be discussed during the meeting or at a
+  subsequent meeting.  Please see Folkson's email for background
+  information about taproot activation mechanisms and a proposed agenda
+  for the meeting.
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [How is the whitepaper decoded from the blockchain?]({{bse}}35959)
+  In a follow up to the original 2015 question, Steven Roose provides a one line
+  `bitcoin-cli` command using `getrawtransaction` against a full node to
+  generate a PDF of the Bitcoin whitepaper. [jb55 provides][bitcoin whitepaper gettxout]
+  a similar command using `gettxout` which works against a pruned node.
+
+- [Full list of "special cases" during Bitcoin Script execution?]({{bse}}101142)
+  Pieter Wuille provides an overview of Bitcoin script evaluation in pseudocode
+  including the conditions for additional rules for both [BIP16][] P2SH and
+  [BIP141][] segwit.
+
+- [Would first-seen prevent a double spend attack?]({{bse}}101827)
+  David Lynch asks if moving away from [Replace-By-Fee (RBF)][topic rbf] would
+  prevent double spend attacks. Pieter Wuille describes a variety of nuanced
+  considerations and incentives surrounding transaction propagation on the
+  network concluding that no type of unconfirmed transaction can be trusted and
+  users should wait for confirmations.
+
+- [How do light clients using compact block filters get relevant unconfirmed transactions?]({{bse}}101512)
+  User Pseudonymous explains that while [BIP37][topic transaction bloom
+  filtering] bloom transaction filtering supports unconfirmed transactions,
+  [compact block filters][topic compact block filters] have no such
+  consideration and apply exclusively to servicing light clients with condensed
+  block data.
```

Comment:
The main thrust of Pseudonymous's answer was that SPV clients _shouldn't_ be displaying unconfirmed transactions to users, since there's no way to verify that they're valid. I think this summary should include that detail.